connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -274,6 +274,19 @@ public Response restartConnector(final @PathParam("connector") String connector,
         return Response.accepted().entity(stateInfo).build();
     }
 
+    @PUT
+    @Path("/{connector}/stop")
+    @Operation(summary = "Stop the specified connector",
+               description = "This operation is idempotent and has no effects if the connector is already stopped")
+    public void stopConnector(
```

Comment:
Good eye. The difference between pause/resume and stop is that the latter may require forwarding to the leader, which (right now) breaks for requests that 1) don't return a body and 2) return a 2xx HTTP status that isn't 204.

This is also why requests to, e.g., delete a connector return 204 instead of 202.

It wouldn't be terribly difficult to fix the `RestClient` class to be able to handle empty-body 202 responses correctly, but since I doubt the inconsistency between the pause/resume and stop endpoints is going to cause serious headaches, I figured we could go with the simpler approach.